src/layout.js

```diff
@@ -44,19 +44,20 @@ export const LAYOUTS = [
 ];
 
 /**
- * @enum {string}
+ * @enum {number}
  */
 export const Layout = {
-  // Naming convention enforced via tests.
-  NODISPLAY: 'nodisplay',
-  FIXED: 'fixed',
-  FIXED_HEIGHT: 'fixed-height',
-  RESPONSIVE: 'responsive',
-  CONTAINER: 'container',
-  FILL: 'fill',
-  FLEX_ITEM: 'flex-item',
-  FLUID: 'fluid',
-  INTRINSIC: 'intrinsic',
+  // Values should match indices of `LAYOUTS`.
+  // Naming convention and indices enforced via tests.
+  NODISPLAY: LAYOUTS.indexOf('nodisplay'),
```

Comment:
Why is this better?

Reply:
Saves a few bytes by having the enum keys be number literals instead of string literals.